README.md

```diff
@@ -3,8 +3,8 @@ HDF5 version 1.15.0 currently under development
 ![HDF5 Logo](doxygen/img/HDF5.png)
 
 [![develop build status](https://img.shields.io/github/actions/workflow/status/HDFGroup/hdf5/main.yml?branch=develop&label=develop)](https://github.com/HDFGroup/hdf5/actions?query=branch%3Adevelop)
-[![oneAPI CMake build status](https://img.shields.io/github/actions/workflow/status/HDFGroup/hdf5/linux-icx.yml?branch=develop&label=oneAPI-C)](https://github.com/HDFGroup/hdf5/actions?query=branch%3Adevelop)
-[![oneAPI Autotools build status](https://img.shields.io/github/actions/workflow/status/HDFGroup/hdf5/linux-icx-auto.yml?branch=develop&label=oneAPI-A)](https://github.com/HDFGroup/hdf5/actions?query=branch%3Adevelop)
+[![oneAPI CMake build status](https://img.shields.io/github/actions/workflow/status/HDFGroup/hdf5/intel-cmake.yml?branch=develop&label=oneAPI-C)](https://github.com/HDFGroup/hdf5/actions?query=branch%3Adevelop)
```

Comment:
Why two badges? They are both Intel. If we break out Intel, why not badges for each platform, win, mac, linux?
Then there is the compilers, and options...

We need to decide what we want the badges to indicate.